examples/mockups/pricing.html

```diff
@@ -0,0 +1,786 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="/dist/css/site.css" />
+    <script src="/dist/js/vendor.js"></script>
+    <script src="/dist/js/site.js"></script>
+  </head>
+  <body id="pricing">
+    
+    <!-- Top Menu -->
+    <div class="ui menu fixed borderless" id="topnav">
+      <div class="ui container">
+
+        <a class="header item" href="">
+          <div class="ui image readthedocs-logo"><img src="./images/logo-wordmark-dark.svg"/></div>
+        </a>
+
+        <a class="item active" href="">Product</a>
+        <a class="item" href="">Pricing</a>
+        <a class="item" href="">Company</a>
+        <a class="item" href="">Resources</a>
+    
+        <div class="right menu">
+          <a class="item" href="">Support</a>
+          <div class="item">
+            <a class="ui button primary">
+              Go to Application
+            </a>
+          </div>
+        </div>
+    
+      </div>
+    </div>
+    <!-- end Top Menu -->
+
+
+    <!-- Contents 
+    ----------------------------------- -->
+
+    <!-- Intro -->
+    <section class="ui vertical segment very padded" id="intro">
+      <div class="ui container center aligned">
+        <h1 class="ui header massive">Intro</h1>
+        <p>
+          Some intro explaining stuff.
```

Comment:
I'm thinking we could have a short paragraph explaining we have two main pricing models _- community and corporates -_ and their main differences.

Reply:
Perhaps we just reduce the vertical space allocated to this block then? I think we can explain the community plans in a sentence or two, and keeping the plans visible above the fold is a benefit.

Reply:
That's true. The paragraph can also be moved below the plans. I don't want to remove too much padding and make the text look clustered with the cards bellow.

Reply:
I moved the text section bellow the cards for now.
Personally, I don't mind having a short explanatory text at the top of the page. It gives context and some visual space to bread before jumping into the denser content. But I think this maybe is something we can come back to later, after we have the final content? For now it might be enough to think on that content with the different options we have for display.

Reply:
That sounds like a plan, it might be less of an issue if the first plans blocks ends up on the shorter side.